src/sas/sasgui/perspectives/fitting/basepage.py

```diff
@@ -3364,7 +3375,13 @@ def _get_paste_helper(self, param, orient_param, content):
                         item[2].Enable(True)
                     if item[2].__class__.__name__ == "ComboBox":
                         if content[name][1] in self.model.fun_list:
-                            fun_val = self.model.fun_list[content[name][1]]
+                            # to fix: fun_list is not a dictionary, but a tuple
+                            # so the following line (commented) will fail
+                            # fun_val = self.model.fun_list[content[name][1]]
+                            # I guess the following should work, but 
+                            # could not test as I don't know when this
+                            # part is used by SasView.
+                            fun_val = self.model.fun_list.index(content[name][1])
```

Comment:
Try copying and pasting parameters for spherical_sld

Reply:
Copying and pasting standard parameters works OK in spherical_sld model, while for the combo box the copy/paste is not really useful --> Should this  if item[2].__class__.__name__ == "ComboBox": block be removed?